swarm/src/behaviour/toggle.rs

```diff
@@ -436,7 +436,7 @@ mod tests {
     ///    [`ConnectionHandler`] might be notified of an inbound upgrade error
     ///    unrelated to its own upgrade logic. For example when nesting a
     ///    [`ToggleProtoHandler`] in a
-    ///    [`ConnectionHandlerSelect`](crate::protocols_handler::ConnectionHandlerSelect)
+    ///    [`ConnectionHandlerSelect`](crate::connections_handler::ConnectionHandlerSelect)
     ///    the former might receive an inbound upgrade error even when disabled.
     ///
     /// [`ToggleProtoHandler`] should ignore the error in both of these cases.
```

Comment:
How about also renaming `ToggleProtoHandler` to `ToggleConnectionHandler` and `ToggleIntoProtoHandler` to  `ToggleIntoConnectionHandler`?

Reply:
> How about also renaming `ToggleProtoHandler` to `ToggleConnectionHandler` and `ToggleIntoProtoHandler` to  `ToggleIntoConnectionHandler`?

Oh I missed these! Thanks for showing me.

Reply:
Done :)